### PR TITLE
#1 ---[Fix] Global emission uses malloc when returned by a function

### PR DESCRIPTION
**Pull request by doru1004**
This patch returns the result of a malloc when global value is returned.

### CODE REVIEW

File: src/Conversion/ONNXToKrnl/Tensor/Constant.cpp [hunk 1]
```diff
@@ -25,6 +39,8 @@ struct ONNXConstantOpLowering : public ConversionPattern {
     auto loc = op->getLoc();
     auto constantOp = llvm::dyn_cast<ONNXConstantOp>(op);
 
+    printf("Operation is returned: %d\n", checkOpResultIsReturned(&constantOp));
+
```
Comment (chentong319) [inline src/Conversion/ONNXToKrnl/Tensor/Constant.cpp:43]: print for debugging only?

Comment (doru1004) [reply src/Conversion/ONNXToKrnl/Tensor/Constant.cpp]: done

**Review by chentong319 — APPROVED** [review]
Can you enable the test for SizeOp in backend (uncomment line 573 in test/backend/test.py)? This patch should make these test go through. Otherwise, the patch is good.

**Comment by doru1004** [comment]
> Can you enable the test for SizeOp in backend (uncomment line 573 in test/backend/test.py)? This patch should make these test go through. Otherwise, the patch is good.

Just enabled them.

**Pull request merged**